packages/react/src/components/ActionList/ActionListGroup.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { forwardRef, useCallback } from 'react';
+import { useId } from 'react-id-generator';
+import {
+  type ActionListSelectionType,
+  type onActionCallbackFunction,
+  type onActionEvent,
+  ActionListProvider,
+  useActionListContext
+} from './ActionListContext';
+import { ListboxGroup } from '../Listbox';
+import { ContentNode } from '../../types';
+
+interface ActionListGroupProps extends React.HTMLAttributes<HTMLLIElement> {
+  /** Provides a label for the group of action items */
+  label: ContentNode;
+
+  /** Limits the amount of selections that can be made within an action group */
+  selectionType?: ActionListSelectionType | null;
```

Comment:
```suggestion
  /** Limits the amount of selections that can be made within an action group */
  selectionType?: ActionListSelectionType
```

Reply:
In this case we want to allow `null` or `undefined`. The problem is if you pass `undefined` into a nested context, it won't override a parent context but `null` will.